{"uid":"cab362ee307b8b89","name":"test_view_deleted_team_returns_404","fullName":"test.teams.test_get_team#test_view_deleted_team_returns_404","historyId":"3fea3e59d795af85e5e11d6bcdacd986","time":{"start":1726090713896,"stop":1726090714255,"duration":359},"status":"skipped","statusMessage":"XFAIL This test case is expected to fail due to known issue.\n\nAssertionError: Expected status code 404, but got 200. Response text: {\"id\":\"66e20dd9d438dfbd4\",\"name\":\"Team test how\",\"deleted\":true,\"positionList\":[\"Designer\",\"Qa\",\"Dev\"],\"createdAt\":\"2024-09-11 21:38:33\",\"modifiedAt\":\"2024-09-11 21:38:34\",\"rolesIds\":[\"52bd3ee937361\",\"52bc41359084d\"],\"rolesNames\":{\"52bd3ee937361\":\"Sales\",\"52bc41359084d\":\"Sales Manager\"},\"layoutSetId\":\"66d8f8ff4f1c23bca\",\"layoutSetName\":\"Layaout 1\",\"workingTimeCalendarId\":\"66d8f90b424428d1b\",\"workingTimeCalendarName\":\"calendar\"}","statusTrace":"setup_teardown_team_module = {'createdAt': '2024-09-11 21:38:33', 'deleted': False, 'id': '66e20dd9d438dfbd4', 'layoutSetId': '66d8f8ff4f1c23bca', ...}\n\n    @pytest.mark.viewteam\n    @pytest.mark.xfail(reason=\"This test case is expected to fail due to known issue.\", condition=True)\n    def test_view_deleted_team_returns_404(setup_teardown_team_module):\n        team = setup_teardown_team_module\n        url = EndpointTeams.get_team_id(team['id'])\n        TeamService.delete_team(url)\n        response = TeamService.view_team(url)\n>       AssertionManager.assert_status_code_404(response)\n\ntest/teams/test_get_team.py:31: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [200]>\n\n    @staticmethod\n    def assert_status_code_404(response):\n        logger.info(f\"Validating status code 404 for response with URL: {response.url}\")\n        try:\n>           assert response.status_code == 404, (\n                f\"Expected status code 404, but got {response.status_code}. Response text: {response.text}\"\n            )\nE           AssertionError: Expected status code 404, but got 200. Response text: {\"id\":\"66e20dd9d438dfbd4\",\"name\":\"Team test how\",\"deleted\":true,\"positionList\":[\"Designer\",\"Qa\",\"Dev\"],\"createdAt\":\"2024-09-11 21:38:33\",\"modifiedAt\":\"2024-09-11 21:38:34\",\"rolesIds\":[\"52bd3ee937361\",\"52bc41359084d\"],\"rolesNames\":{\"52bd3ee937361\":\"Sales\",\"52bc41359084d\":\"Sales Manager\"},\"layoutSetId\":\"66d8f8ff4f1c23bca\",\"layoutSetName\":\"Layaout 1\",\"workingTimeCalendarId\":\"66d8f90b424428d1b\",\"workingTimeCalendarName\":\"calendar\"}\n\ncore/assertions/status_code_assertion.py:61: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1726090705999,"stop":1726090706016,"duration":17},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"setup_teardown_team_module","time":{"start":1726090713713,"stop":1726090713895,"duration":182},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"status":"skipped","statusMessage":"XFAIL This test case is expected to fail due to known issue.\n\nAssertionError: Expected status code 404, but got 200. Response text: {\"id\":\"66e20dd9d438dfbd4\",\"name\":\"Team test how\",\"deleted\":true,\"positionList\":[\"Designer\",\"Qa\",\"Dev\"],\"createdAt\":\"2024-09-11 21:38:33\",\"modifiedAt\":\"2024-09-11 21:38:34\",\"rolesIds\":[\"52bd3ee937361\",\"52bc41359084d\"],\"rolesNames\":{\"52bd3ee937361\":\"Sales\",\"52bc41359084d\":\"Sales Manager\"},\"layoutSetId\":\"66d8f8ff4f1c23bca\",\"layoutSetName\":\"Layaout 1\",\"workingTimeCalendarId\":\"66d8f90b424428d1b\",\"workingTimeCalendarName\":\"calendar\"}","statusTrace":"setup_teardown_team_module = {'createdAt': '2024-09-11 21:38:33', 'deleted': False, 'id': '66e20dd9d438dfbd4', 'layoutSetId': '66d8f8ff4f1c23bca', ...}\n\n    @pytest.mark.viewteam\n    @pytest.mark.xfail(reason=\"This test case is expected to fail due to known issue.\", condition=True)\n    def test_view_deleted_team_returns_404(setup_teardown_team_module):\n        team = setup_teardown_team_module\n        url = EndpointTeams.get_team_id(team['id'])\n        TeamService.delete_team(url)\n        response = TeamService.view_team(url)\n>       AssertionManager.assert_status_code_404(response)\n\ntest/teams/test_get_team.py:31: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [200]>\n\n    @staticmethod\n    def assert_status_code_404(response):\n        logger.info(f\"Validating status code 404 for response with URL: {response.url}\")\n        try:\n>           assert response.status_code == 404, (\n                f\"Expected status code 404, but got {response.status_code}. Response text: {response.text}\"\n            )\nE           AssertionError: Expected status code 404, but got 200. Response text: {\"id\":\"66e20dd9d438dfbd4\",\"name\":\"Team test how\",\"deleted\":true,\"positionList\":[\"Designer\",\"Qa\",\"Dev\"],\"createdAt\":\"2024-09-11 21:38:33\",\"modifiedAt\":\"2024-09-11 21:38:34\",\"rolesIds\":[\"52bd3ee937361\",\"52bc41359084d\"],\"rolesNames\":{\"52bd3ee937361\":\"Sales\",\"52bc41359084d\":\"Sales Manager\"},\"layoutSetId\":\"66d8f8ff4f1c23bca\",\"layoutSetName\":\"Layaout 1\",\"workingTimeCalendarId\":\"66d8f90b424428d1b\",\"workingTimeCalendarName\":\"calendar\"}\n\ncore/assertions/status_code_assertion.py:61: AssertionError","steps":[],"attachments":[{"uid":"dc821309525630f1","name":"log","source":"dc821309525630f1.txt","type":"text/plain","size":5847},{"uid":"91fcd81a78bedda9","name":"stderr","source":"91fcd81a78bedda9.txt","type":"text/plain","size":6136}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":2,"hasContent":true},"afterStages":[{"name":"setup_teardown_team_module::0","time":{"start":1726090714285,"stop":1726090714462,"duration":177},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"},{"name":"tag","value":"viewteam"},{"name":"parentSuite","value":"test.teams"},{"name":"suite","value":"test_get_team"},{"name":"host","value":"fv-az837-91"},{"name":"thread","value":"2054-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test.teams.test_get_team"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["viewteam","@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"]},"source":"cab362ee307b8b89.json","parameterValues":[]}